docs/guides/VTEX-Platform-Overview/store-architecture/store-architecture-use-cases.md

```diff
@@ -0,0 +1,163 @@
+
+### Learn more
+
+- [Unified Commerce Strategies](https://help.vtex.com/en/tracks/estrategias-de-comercio-unificado--3WGDRRhc3vf1MJb9zGncnv/2LGAiUnHES1enjHsfi8fI3)
+
+## Multi-language and Multi-currency
+
+Some of our customers are present in multiple countries and want to have every local store running on VTEX. We provide a multi-language and multi-currency architecture to serve this customer, as you can see below.
+
+### Single account, multi-binding
+
+In this architecture, a single VTEX account is configured to support multiple languages and currencies, using a [multidomain](https://help.vtex.com/en/tracks/vtex-store-overview--eSDNk26pdvemF3XKM0nK9/4yPqZQyj0t675QpcG7H6yl#multistore) approach.
+
+The reference architecture below exemplifies its main characteristics:
+
+- **Separated website**: One single account, but each store with its own domain, bound to different [sales channels](https://help.vtex.com/en/tracks/vtex-store-overview--eSDNk26pdvemF3XKM0nK9/4EPwTXx5oFdSG1dA3zIchz#trade-policy) by [bindings](https://help.vtex.com/en/tutorial/what-is-binding--4NcN3NJd0IeYccgWCI8O2W?&utm_source=autocomplete).  
+- **Search**: [Settings](https://developers.vtex.com/docs/guides/search-overview) are the same for all websites.
+- **CMS**: Shared access to [Content Management System (CMS)](https://developers.vtex.com/docs/guides/vtex-io-documentation-cms) between websites.
+- **Customer Data Storage**: One single [Master Data](https://developers.vtex.com/docs/guides/master-data-introduction) for all stores.
+- **Catalog**: One single catalog, segmented by sales channels.
+- **Logistics**: Each store has its logistics operation managed by different warehouses in the same Logistics panel.
+- **Order Management System (OMS)**: One single Logistics and OMS for all stores.
+- **Payments**: One Payments panel for all stores.
+- **Prices and Promotions**: It is possible to configure different prices and promotions for each store, as you can segment by the sales channel. But prices and promotions for all stores are managed in the same panel.
+- **Message Center**: One single [Message Center](https://help.vtex.com/en/tutorial/conhecendo-o-message-center--tutorials_84) for all stores.
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/multi-currency-language-single-account.png)
+
+#### Why choose this architecture?
+
+- Supports [multi-currency](https://developers.vtex.com/docs/guides/vtex-io-documentation-internationalizing-product-prices) natively.
+- Supports multi-language with [Messages](https://developers.vtex.com/docs/guides/storefront-content-internationalization) and [Catalog Translation App](https://developers.vtex.com/docs/guides/catalog-internationalization).
```

Comment:
Based on these topics, I think only stores on Store Framework can operate in this architecture, right? Maybe it's worth mentioning that in a callout in the beginning of this section